libs/README.md

```diff
@@ -1,6 +1,7 @@
 # Dependency Management in ILIAS
 
 The are currently three type of external libraries:
-- PHP dependencies installed using composer in /libs/composer, see [composer-readme](libs/composer/README.md).
+- PHP dependencies installed using composer in /libs/composer, see [composer-readme](composer/README.md).
```

Comment:
These changes can be ignored in this PR since the dependency-management using bower will be part of another PR